coalib/coala_main.py

```diff
@@ -69,6 +72,16 @@ def run_coala(log_printer=None,
         dtag = str(sections['default'].get('dtag', None))
         config_file = os.path.abspath(str(sections["default"].get("config")))
 
+        settings_hash = get_settings_hash(sections)
+        flush_cache = sections['default'].get('flush_cache', False)
+        if not flush_cache and settings_changed(log_printer, settings_hash):
```

Comment:
@sils1297 we'd need to recompute it again at https://github.com/coala-analyzer/coala/pull/2016/files#diff-bf856c6981ec06ee90cafbcc45a41874R113